src/components/dataDisplays/HoustonSightingsDisplay.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
This is set up to be used my `SightingsCard`. I am trying to do separate PRs so the feature doesn't get slowed down by the cleanup